instance.tf

```diff
@@ -0,0 +1,45 @@
+data "aws_ami" "amzn-ami" {
+	most_recent = true
+	filter{
+	  name = "name"
+	  values = ["amzn2-ami-hvm-*-gp2"]
+	}
+	
+	owners = ["137112412989"] #AWS
+}
+
+resource "aws_security_group" "allow_tls" {
+	name = "allow_tls"
+	description = "Allow TLS inbound traffic"
+        vpc_id = aws_vpc.core-infra.id
+
+	ingress {
+	  description = "TLS from VPC"
+	  from_port = 80
+	  to_port = 22
+	  protocol = "tcp"
+	}
+
+	egress {
+	  from_port = 22
```

Comment:
Create a new security group call `egress-all` with the following, and remove line 16 to line 21
```
  "egress" {
    cidr_blocks = ["0.0.0.0/0"]
    from_port   = 0
    protocol    = "-1"
    to_port     = 0
  }
```